tests/yam/agama/auto/playwright_test.pm

```diff
@@ -0,0 +1,22 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run playwright tests directly from the Live ISO.
```

Comment:
Could you please also prepend this sentence? "Run interactive installation with Agama", the details how this is done is not so important, it might change in the future, but your sentence could be also useful at this state, we can have those two lines.

Reply:
updated